blob/helper.go

```diff
@@ -11,21 +11,8 @@ import (
 	"github.com/celestiaorg/celestia-node/share"
 )
 
-// SharesToBlobs takes raw shares and converts them to the blobs.
-func SharesToBlobs(rawShares []share.Share) ([]*Blob, error) {
-	if len(rawShares) == 0 {
-		return nil, ErrBlobNotFound
-	}
-
-	appShares, err := toAppShares(rawShares...)
-	if err != nil {
-		return nil, err
-	}
-	return parseShares(appShares)
-}
-
 // parseShares takes shares and converts them to the []*Blob.
-func parseShares(appShrs []shares.Share) ([]*Blob, error) {
+func parseShares(appShrs []shares.Share, indexes []int) ([]*Blob, error) {
```

Comment:
why take `[]int` for indexes ? This feels complicated / unreadable

Reply:
`appShrs []shares.Share` can contain multiple blobs, so we need support multiple indexes

Reply:
But I don't have any use case where I need multiple indexes, so I think it is fine to assume that parseShares returns single blob instead []Blob